frontends/web/src/routes/account/summary/accountssummary.tsx

```diff
@@ -289,6 +290,12 @@ class AccountsSummary extends Component<Props, State> {
   public render() {
     const { t, accounts } = this.props;
     const { exported, data, balances } = this.state;
+
+    // remove trailing zeroes for BTC fiat total
+    if (data && data.formattedChartTotal) {
+      data.formattedChartTotal = removeTrailingZeroes(data.formattedChartTotal, data.chartFiat);
```

Comment:
This mutates the data object in the component state, as data is only a reference to the object in the state object, and if `data.formattedChartTotal = …` is reassigned it will mutate also this.state.data.formattedChartTotal (iirc).

Either do this check and formatting after receiving the data in `accountApi.getSummary()` before setState, or alternatively (maybe simpler) :

```diff
diff --git a/frontends/web/src/routes/account/summary/accountssummary.tsx b/frontends/web/src/routes/account/summary/accountssummary.tsx
index f59acb38..63b1c4bb 100644
--- a/frontends/web/src/routes/account/summary/accountssummary.tsx
+++ b/frontends/web/src/routes/account/summary/accountssummary.tsx
@@ -290,10 +290,11 @@ class AccountsSummary extends Component<Props, State> {
   public render() {
     const { t, accounts } = this.props;
     const { exported, data, balances } = this.state;
+    let formattedChartTotal = data?.formattedChartTotal;
 
     // remove trailing zeroes for BTC fiat total
-    if (data && data.formattedChartTotal) {
-      data.formattedChartTotal = removeTrailingZeroes(data.formattedChartTotal, data.chartFiat);
+    if (data && formattedChartTotal) {
+      formattedChartTotal = removeTrailingZeroes(formattedChartTotal, data.chartFiat);
     }
```